frontend/package.json

```diff
@@ -35,8 +36,9 @@
     "autoprefixer": "^7.2.6",
     "babel-core": "^6.26.0",
     "babel-eslint": "^8.2.2",
-    "babel-jest": "^22.4.1",
-    "babel-loader": "^7.1.3",
+    "babel-jest": "^22.4.3",
+    "babel-loader": "^7.1.4",
+    "babel-plugin-lodash": "^3.3.2",
```

Comment:
Do we actually need that? Wouldn't be better to replace all `import _ from 'lodash'` with just import the function that we need?

Reply:
I prefer this for two reasons:
- this is equivalent, and allows us to not have to worry about that
- this will most likely be more reliable at that task than us

less busy and lazier developers is what the world needs ;)

Reply:
Fine, but let's keep enforcing individual imports

Reply:
Hehe ok, I just always try to minimalize the number of dependencies.

Reply:
indeed, but I am more concerned about dependencies that make it to the user's browser, and not so much with serverside stuff